accepted/2021/statics-in-interfaces/README.md

```diff
@@ -6,8 +6,1267 @@
+        : IBinaryFloatingPoint<double>,
+          IConvertible,
+          IFixedWidth<double>
+    {
+    }
+
+    public struct Enum
+        : IComparable,
+          IConvertible,
+          IFormattable
+    {
+    }
+
+    public struct Guid
+        : IComparableOperators<Guid, Guid>,
+          ISpanFormattable,
+          ISpanParseable<Guid>
+    {
+    }
+
+    public struct Half
+        : IBinaryFloatingPoint<Half>,
+          IFixedWidth<Half>
+    {
+    }
+
+    public struct Int16
+        : IBinaryInteger<short>,
+          IConvertible,
+          IFixedWidth<short>,
+          ISignedNumber<short>
+    {
+    }
+
+    public struct Int32
+        : IBinaryInteger<int>,
+          IConvertible,
+          IFixedWidth<int>,
+          ISignedNumber<int>
+    {
+    }
+
+    public struct Int64
+        : IBinaryInteger<long>,
+          IConvertible,
+          IFixedWidth<long>,
+          ISignedNumber<long>
+    {
+    }
+
+    public struct IntPtr
+        : IBinaryInteger<nint>,
+          IFixedWidth<nint>,
+          ISerializable,
+          ISignedNumber<nint>
+    {
+    }
+
+    public struct SByte
+        : IBinaryInteger<sbyte>,
+          IConvertible,
+          IFixedWidth<sbyte>,
+          ISignedNumber<sbyte>
+    {
+    }
+
+    public struct Single
+        : IBinaryFloatingPoint<float>,
+          IConvertible,
+          IFixedWidth<float>
+    {
+    }
+
+    public struct TimeOnly
+        : IComparableOperators<TimeOnly, TimeOnly>,
+          IFixedWidth<TimeOnly>,
+          ISpanFormattable,
+          ISpanParseable<TimeOnly>,
+          ISubtractable<TimeOnly, TimeOnly, TimeSpan>
+    {
+    }
+
+    public struct TimeSpan
+        : IAddable<TimeSpan, TimeSpan, TimeSpan>,
+          IComparableOperators<TimeSpan, TimeSpan>,
+          IDivisable<TimeSpan, double, TimeSpan>,
+          IDivisable<TimeSpan, TimeSpan, double>,
+          IFixedWidth<TimeSpan>,
+          IMultipliable<TimeSpan, double, TimeSpan>,
+          INegatable<TimeSpan, TResult>,
+          ISpanFormattable,
+          ISubtractable<TimeSpan, TimeSpan, TimeSpan>
+    {
+    }
+
+    public struct UInt16
+        : IBinaryInteger<ushort>,
+          IConvertible,
+          IFixedWidth<ushort>,
+          IUnsignedNumber<ushort>
+    {
+    }
+
+    public struct UInt32
+        : IBinaryInteger<uint>,
+          IConvertible,
+          IFixedWidth<uint>,
+          IUnsignedNumber<uint>
+    {
+    }
+
+    public struct UInt64
+        : IBinaryInteger<ulong>,
+          IConvertible,
+          IFixedWidth<ulong>,
+          IUnsignedNumber<ulong>
+    {
+    }
+
+    public struct UIntPtr
+        : IBinaryInteger<nuint>,
+          IFixedWidth<nuint>,
+          ISerializable,
+          IUnsignedNumber<nuint>
+    {
+    }
+}
+```
+
+### Pending Concepts
+
+There are several types which may benefit from some interface support. These include, but aren't limited to:
+* System.Array
+* System.DateOnly
```

Comment:
You already included this one above.  Same with TimeOnly.

Reply:
Forgot to remove them from this list when I added them above. Will fix.